racket/src/cs/core.sls

```diff
@@ -641,11 +649,11 @@
   (include "core/extfl.ss")
   (include "core/place.ss")
   (include "core/foreign.ss")
-  (include "core/future.ss")	
-  (include "core/queue.ss")
-  (include "core/future-scheduler.ss")
+  (include "core/lock.ss")
+  (include "core/future.ss")
   
   (set-no-locate-source!)
+  ;; with this below line enabled, errors cause hangs
```

Comment:
Can you make this comment more specific? As-is, it seems to say there's a bug in the implementation, but I think you mean that (other) bugs in the implementation are harder to debug with execptions set up normally.